lib/menu_pages/items_page.dart

```diff
@@ -102,16 +98,16 @@ class _ItemsPageState extends State<ItemsPage> {
           ),
         ),
       ),
-      Positioned(
-        bottom: 16,
-        right: 16,
-        child: FloatingActionButton(
-          child: Icon(Icons.add),
-          onPressed: () {
-            _openPopup(context);
-          },
-        ),
-      )
+      // Positioned(
+      //   bottom: 16,
+      //   right: 16,
+      //   child: FloatingActionButton(
+      //     child: Icon(Icons.add),
+      //     onPressed: () {
+      //       openAddItemPopup(context);
+      //     },
+      //   ),
+      // ),
```

Comment:
@ahamzatariq , Remove commented code.

Reply:
It's already removed in the later commits.